framework/include/utils/RankTwoTensorImplementation.h

```diff
@@ -1033,26 +1033,27 @@ RankTwoTensorTempl<T>::initRandom(unsigned int rand_seed)
 
 template <typename T>
 RankTwoTensorTempl<T>
-RankTwoTensorTempl<T>::genRandomTensor(T scale, T offset)
+RankTwoTensorTempl<T>::genRandomTensor(T stddev, T mean)
 {
   RankTwoTensorTempl<T> result;
   for (const auto i : make_range(N))
     for (const auto j : make_range(N))
-      result(i, j) = (MooseRandom::rand() + offset) * scale;
+      result(i, j) = (MooseRandom::rand() + mean) * stddev;
   return result;
 }
 
 template <typename T>
 RankTwoTensorTempl<T>
-RankTwoTensorTempl<T>::genRandomSymmTensor(T scale, T offset)
+RankTwoTensorTempl<T>::genRandomSymmTensor(T stddev, T mean)
 {
-  auto r = [&]() { return (MooseRandom::rand() + offset) * scale; };
+  auto r = [&]() { return (MooseRandom::rand() + mean) * stddev; };
```

Comment:
`MooseRandom::rand()` returns a uniform distribution over [0,1), so the new parameter names are not correct.

Reply:
Yeah you're right. 

I give up on the documentation of these two methods... I don't want to lie before we fix them.